IDL/member_id.idl

```diff
@@ -0,0 +1,128 @@
+@mutable
+struct FixId
+{
+    @id(100)
+    octet o;
+    @id(300)
+    short s;
+    @id(500)
+    long l;
+    long long ll;
+};
+
+@mutable
+struct FixHashidDefault
+{
+    @hashid
+    octet o;
+    @hashid
+    short s;
+    @hashid
+    long l;
+    long long ll;
+};
+
+@mutable
+struct FixHashid
+{
+    @hashid("octet")
+    octet o;
+    @hashid("short")
+    short s;
+    @hashid("long")
+    long l;
+    long long ll;
+};
+
+@mutable
+struct FixMix
+{
+    @id(100)
+    octet o;
+    @hashid
+    short s;
+    @hashid("long")
+    long l;
+    long long ll;
+};
+
+@mutable
+@autoid
+struct AutoidDefault
+{
+    char c;
+    @id(100)
+    octet o;
+    @hashid
+    short s;
+    @hashid("long")
+    long l;
+    long long ll;
+};
+
+@mutable
+@autoid(SEQUENTIAL)
+struct AutoidSequential
+{
+    char c;
+    @id(100)
+    octet o;
+    @hashid
+    short s;
+    @hashid("long")
+    long l;
+    long long ll;
+};
+
+@mutable
+@autoid(HASH)
+struct AutoidHash
+{
+    char c;
+    @id(100)
+    octet o;
+    @hashid
+    short s;
+    @hashid("long")
+    long l;
+    long long ll;
+};
+
+@autoid
+struct DerivedAutoidDefault : AutoidDefault
+{
+    char cd;
+    @id(101)
+    octet od;
+    @hashid
+    short sd;
+    @hashid("long2")
+    long ld;
+    long long lld;
+};
+
+@autoid(SEQUENTIAL)
+struct DerivedAutoidSequential : AutoidSequential
+{
+    char cd;
+    @id(101)
+    octet od;
+    @hashid
+    short sd;
+    @hashid("long2")
+    long ld;
+    long long lld;
+};
+
+@autoid(HASH)
+struct DerivedAutoidHash : AutoidHash
+{
+    char cd;
+    @id(101)
+    octet od;
+    @hashid
+    short sd;
+    @hashid("long2")
+    long ld;
+    long long lld;
+};
```

Comment:
There is a specific inheritance test case that it would be nice to test as well. What happens if there is nested inheritance as the following IDL:

```
struct SomeStruct
{
    short some_var;
};

struct EmptyStruct : SomeStruct
{
};

struct AnotherInheritedStruct : EmptyStruct
{
    long another_var;
};
```

Probably the logic in IDL Parser would handle this case correctly, but I suggest having the test case to ensure that in the future is not broken.

Reply:
Done in 65376cd